src/megaapi_impl.cpp

```diff
@@ -31928,7 +31928,7 @@ string MegaPushNotificationSettingsPrivate::generateJson() const
         json.append(",");
     }
 
-    if (mGlobalChatsDND > -1)
+    if (isGlobalChatsDndEnabled())
```

Comment:
Not sure if it's done in purpose, but previously it used to test whether the global DND for chats was unset. Now, it checks it's effective (note the DND's ts can already be in the past and this method will return false, where the previous method will test successfully (true).
Maybe it's better this way, since the DND would be removed. But take into account it changes the existing logic, and you may want to keep it untouched. If you are unsure, @jgandres , please, check with Alber. Thanks

Reply:
We have been talking about this topic, and we don't see any case we need to preserve an expired DND, If you know any case please tell me in order to make the adjustment.

Reply:
I don't think so either. So... all fine ;)